### DISCUSSION TITLE
widget views should exit with one 'back'

### DISCUSSION THREAD

**Discussion by AeliusSaionji** [TechbeeAT/jtxBoard#25] [Ideas]
Title says it all.

At the moment, when you open jtxBoard from the widget, pressing back breaks "flow" and summons a jtxBoard page. The expected behavior should instead return you to the previous context- the android launcher.

Thanks!

**Comment by patrickunterwegs** [comment]
Hey, thanks, yeah, that's a bit bothering, I just fixed it, that will be in 2.1.3!

**Reply by TheBearDidLady** [reply]
I currently can't open jtxBoard itself from the widget, only an existing item or the add new item screen, so I've been using this as a workaround.

**Reply by patrickunterwegs** [reply]
@TheBearDidLady what error message do you get?
Is this related to the basic discussion about the back button of this topic here?

**Reply by TheBearDidLady** [reply]
Sorry, newbie error. Should have made the connection clearer. (Newbie both to GitHub and to jtxBoard).

No error message. I hadn't found a way of opening the app from the widget, only opening a specific item or creating a new item. I had been using the incorrect back button behaviour described above to get to the full app, so with that behaviour changing, I'd lose the workaround. However, it turns out it was my own fat finger error anyway. I had been clicking on the "Tasks" heading within the widget rather than the jtx icon to the left of that heading.

**Reply by patrickunterwegs** [reply]
Alight, no worries ;-)
Ok, I See, so actually it makes sense to extend the clickable area to open the app also to the text next to the logo. I'll check that! :-)
